src/common/pf_snmp.c

```diff
@@ -142,27 +217,14 @@ void pf_snmp_get_system_name (pnet_t * net, pf_snmp_system_name_t * name)
    error = pf_file_load (directory, PF_FILENAME_SYSNAME, name, sizeof (*name));
    if (error)
    {
-      error = pnal_get_hostname (name->string);
-      if (error)
+      int result = pnal_get_hostname (name->string);
```

Comment:
This stack puts variable definitions in the beginning of functions.

Reply:
Done.